### PR TITLE
Remove payout timing notice and update the help tooltip.

### LINKED ISSUE #7900: Localize Calendar Days vs Business Days Language in the Deposits Section of the Payments > Overview Page
### Description

Update: per the [discussion](https://github.com/Automattic/woocommerce-payments/issues/7900#issuecomment-2481494581) below, we are going to remove the notice.

On the Payments > Overview Page, in the Deposits section, EU Accounts say "It may taken 1-3 business days for deposits to reach your bank account." but only the US, Australia, Japan, New Zealand, and the UAE use Business Days. Other countries should show Calendar Days here. 

Markup on 2023-12-13 at 11:08:14 AM

### Acceptance criteria
* Have an Account not based in the US, Australia, Japan, New Zealand, or the UAE. (The screenshot above is from a German Account). 
* Check the Deposits Section of the Payments > Overview page.

### PR DESCRIPTION
**Pull request by shendy-a8c**
Fixes https://github.com/Automattic/woocommerce-payments/issues/7900

#### Changes proposed in this Pull Request

Note: the changes from this PR looks different from what the original issue https://github.com/Automattic/woocommerce-payments/issues/7900 explains because there's some discussions there. If there's further discussion, I'll update this PR accordingly.

On the Payments overview page:
- Remove the payout timing notice.

386996435-9d456c02-18cc-4284-b24b-9c0b0686cabf

- Update the help tooltip.

389945513-1297c974-a633-45f1-a1a2-4ce377efecbd

#### Testing instructions

* Go to Payments overview page.
* Make sure payout notice is not there anymore.
* Make sure help tooltip is updated according to the discussion happening in https://github.com/Automattic/woocommerce-payments/issues/7900.

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `9812` or branch name `update/7900-payout-notice` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=update/7900-payout-notice) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 04e4c662b4a46665a1e52aa380743e199385df0b
- Build time: 2024-12-02 21:56:01 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: client/components/deposits-overview/deposit-notices.tsx [hunk 1]
```diff
@@ -104,22 +104,6 @@ export const NewAccountWaitingPeriodNotice: React.FC = () => (
 	</InlineNotice>
 );
 
-/**
- * Renders a notice informing the user of the number of days it may take for deposits to appear in their bank account.
- */
-export const DepositTransitDaysNotice: React.FC = () => (
-	<InlineNotice
-		icon={ tip }
```
Comment (brucealdridge) [inline client/components/deposits-overview/deposit-notices.tsx:112]: Tip isn't used anymore and can be removed from the import.

**Review by brucealdridge — APPROVED** [review]
image

✅ Changes look good and match with the recommendations in the issue.
The js linting suggests that `import { tip } from '@wordpress/icons';` can be removed from `deposit-notices.tsx`

File: client/components/deposits-overview/deposit-notices.tsx (hunk 1 above)
Comment (shendy-a8c) [reply client/components/deposits-overview/deposit-notices.tsx:112]: Good catch! Removed in [a14783e](https://github.com/Automattic/woocommerce-payments/pull/9812/commits/a14783e73348204b9f148415a68ce3ab1270a207).

**Comment by shendy-a8c** [comment]
I updated the copy per last comment https://github.com/Automattic/woocommerce-payments/issues/7900#issuecomment-2506489574.

391726712-b26d807f-cc79-4af0-ab4a-e65d68edeeba

**Pull request merged**